samples/Exporters/Console/Exporters.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
this is an exiting project right?

Reply:
probablky a merge issue... The "exporters" folder is gone.

Reply:
Fixed. Yes this was a part of merge issue.